### ISSUE TITLE
Accessibility: Underline all hyperlinks on hover

### ISSUE THREAD

**Issue by astridu** [sul-dlss/happy-heron#1752]
Issue: Furthermore, color should not be the only visual cue. An additional visual cue could be underlining the link.

**Comment by jcoyne** [comment]
@astridu how do we square this requirement with the identity guide / Decanter which does not underline links: https://decanter.stanford.edu/

**Comment by astridu** [comment]
I'm not sure decanter is up to date. However, the underline requirement (we could also bold, but that seems more visually distracting) from SODA is mandatory to be AA compliant.

**Comment by jcoyne** [comment]
Even the stanford homepage doesn't use underlined links: https://www.stanford.edu/

**Comment by astridu** [comment]
I think it's out of date, but I sent a message to the Accessibility Slack channel to verify.

**Comment by jcoyne** [comment]
It sounds like this is true for links that are embedded within text, but probably isn't necessary where our tables make it clear these are links. (see https://usability.yale.edu/web-accessibility/articles/links#designing-links)

Perhaps just having the underline appear on mouseover is enough.

**Comment by astridu** [comment]
Given that we have the option to have the underline appear on hover and not all the time, I would prefer this approach.
